demos/linearlite/server.ts

```diff
@@ -0,0 +1,203 @@
+import express from 'express'
+import bodyParser from 'body-parser'
+import cors from 'cors'
+import pg from 'pg'
+import {
+  ChangeSet,
+  changeSetSchema,
+  CommentChange,
+  IssueChange,
+  ChangeResponse,
+  RowChange,
+} from './src/utils/changes'
+
+const DATABASE_URL = process.env.DATABASE_URL
+
+const { Client } = pg
+const client = new Client(DATABASE_URL)
+client.connect()
+
+const app = express()
+
+app.use(bodyParser.json())
+app.use(cors())
+
+app.get('/', async (_req, res) => {
+  const result = await client.query(
+    "SELECT 'ok' as status, version() as postgres_version, now() as server_time"
+  )
+  res.send(result.rows[0])
+})
+
+app.post('/apply-changes', async (req, res) => {
+  const content = req.body
+  let parsedChanges: ChangeSet
+  try {
+    parsedChanges = changeSetSchema.parse(content)
+    // Any additional validation of the changes can be done here!
+  } catch (error) {
+    console.error(error)
+    res.status(400).send('Invalid changes')
+    return
+  }
+  const changeResponse = await applyChanges(parsedChanges)
+  res.send(changeResponse)
+})
+
+app.listen(3001, () => {
+  console.log('Server is running on port 3001')
+})
+
+async function applyChanges(changes: ChangeSet): Promise<ChangeResponse> {
+  const { issues, comments } = changes
```

Comment:
Perhaps add a comment here saying something like: `// Here you would authenticate the user and validate the changes and check that the changes aren't overwriting anything (LWW)`